oh-my-zsh.sh

```diff
@@ -64,19 +64,21 @@ if [ -z "$ZSH_COMPDUMP" ]; then
   ZSH_COMPDUMP="${ZDOTDIR:-${HOME}}/.zcompdump-${SHORT_HOST}-${ZSH_VERSION}"
 fi
 
-if [[ $ZSH_DISABLE_COMPFIX != true ]]; then
-  # If completion insecurities exist, warn the user without enabling completions.
-  if ! compaudit &>/dev/null; then
-    # This function resides in the "lib/compfix.zsh" script sourced above.
-    handle_completion_insecurities
-  # Else, enable and cache completions to the desired file.
+
+if [[ $ZSH_DISABLE_COMPINIT != true ]]; then
+  if [[ $ZSH_DISABLE_COMPFIX != true ]]; then
+      # If completion insecurities exist, warn the user without enabling completions.
```

Comment:
Yeah, 2-space indents are [standard for Oh My Zsh](https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide#indentation), and match the rest of this file.